scipy/special/tests/test_zeta.py

```diff
@@ -47,3 +49,89 @@ def test_riemann_zeta_avoid_overflow():
     s = -260.00000000001
     desired = -5.6966307844402683127e+297  # Computed with Mpmath
     assert_allclose(sc.zeta(s), desired, atol=0, rtol=5e-14)
+
+
+@pytest.mark.parametrize(
+    "z, desired",
+    # Reference cases taken from mpmath with the script:
```

Comment:
```suggestion
    # # Reference cases taken from mpmath with the script:
```

Reply:
Ah, good thinking. Otherwise someone might accidentally paste in the top line here.